cluster-autoscaler/clusterstate/clusterstate.go

```diff
@@ -896,6 +896,9 @@ func isNodeStillStarting(node *apiv1.Node) bool {
 			return true
 		}
 	}
+	if !kube_util.IsNodeExtendedReady(node) {
```

Comment:
simplify, just:
```
return !kube_util.IsNodeExtendedReady(node)
```

Reply:
changed

Reply:
This means more merge conflicts potentially, although this part isn't changed that often.

Reply:
Hmm, good point. Then I will revert it to cause less problems in the future.